README.md

```diff
@@ -440,51 +434,53 @@ the corresponding transaction information, based on the specified hash value.
 
 Request information about our deployment transaction:
 ```
-curl -d '{ "jsonrpc": "2.0", "id": 1, "method": "getrawtransaction", "params": ["5ec9afdcd30b6e38280df690fb0d80490fdcd94af0c2c0bcd3f47b13114f7175", 1] }' localhost:20331 | json_pp
+curl -d '{ "jsonrpc": "2.0", "id": 1, "method": "getrawtransaction", "params": ["3cdafad701018c6dfbd4f0703bbe4ee2a9828a427cbae15696325907ad402c09", 1] }' localhost:20331 | json_pp
 ```
 
 Where
 - `"jsonrpc": "2.0"` is protocol version
 - `"id": 1` is id of current request
 - `"method": "getrawtransaction"` is requested method
-- `"params": ["5ec9afdcd30b6e38280df690fb0d80490fdcd94af0c2c0bcd3f47b13114f7175", 1]` is an array of parameters, where
-   - `5ec9afdcd30b6e38280df690fb0d80490fdcd94af0c2c0bcd3f47b13114f7175` is deployment transaction hash
+  b8d9b23dbf940cfc54a631c0b177b1caf9f38bfb
+- `"params": ["3cdafad701018c6dfbd4f0703bbe4ee2a9828a427cbae1569632590
+  b8d9b23dbf940cfc54a631c0b177b1caf9f38bfb7ad402c09", 1]` is an array of parameters, where
```

Comment:
Not sure this string wrap displays nicely.

Reply:
Fixed.